packages/flutter/lib/src/material/page_transitions_theme.dart

```diff
@@ -270,6 +270,7 @@ class _ZoomPageTransition extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    final Color backgroundColor = Theme.of(context).canvasColor;
```

Comment:
Why `canvasColor` over `primaryColor` or `scaffoldBackgroundColor`?

Reply:
Great question!

I never considered `primaryColor` because I see it as a color with a much more "major"  role that the other options which could lead to harder migration for people with existing transitions impacted by this change.

I did however consider changing it to `scaffoldBackgroundColor` as they serve very similiar purposes and are more obscure properties but ended up keeping the `canvasColor`. However, it's a change I would totally go for if anyone else has the same opinion.

Reply:
Yes very good point.
After discussing with @TahaTesser, he mentionned that `scaffoldBackgroundColor` will be deprecated, see https://github.com/flutter/flutter/issues/91772.
One can also notice that `canvasColor` is also part of this list. After reading https://github.com/flutter/flutter/pull/116350, I think we should code with `colorScheme.background`.

Reply:
Will do!

Reply:
Just a quick note, I'm going to use `colorScheme.surface` as `colorScheme.background` is also deprecated:

![image](https://github.com/user-attachments/assets/be670d9c-5f9b-48b4-8ce9-11985d2aed79)

https://api.flutter.dev/flutter/material/ColorScheme/background.html